lte/protos/enodebd.proto

```diff
@@ -95,6 +95,26 @@ message SingleEnodebStatus {
   StatusProperty rf_tx_desired = 13;
 }
 
+message DownloadRequest {
+  string file_type = 1;
+  string url = 2;
+  string user_name = 3;
+  string password = 4;
+  string file_size = 5;
+  string target_file_name = 6;
+  uint32 delay_seconds = 7;
+  string success_url = 8;
+  string failure_url = 9;
+  string device_serial = 10;
+  string md5 = 11;
+}
+
+message DownloadResponse {
+  uint32 status = 1;
```

Comment:
What does each number indicate for the status? This should be well documented, and nobody should need to dig through the code to figure this out.